week-1/L-functions-nested/exercise2.js

```diff
@@ -0,0 +1,20 @@
+var mentor1 = "Daniel";
+var mentor2 = "Irina";
+var mentor3 = "Mimi";
+var mentor4 = "Rob";
+var mentor5 = "Yohannes";
+var mentor1 = "Daniel";
+var mentor2 = "Irina";
+var mentor3 = "Mimi";
+var mentor4 = "Rob";
+var mentor5 = "Yohannes";
+
+var mentors = ["Daniel", "Irina", "Mimi", "Rob", "Yohannes"];
+console.log(mentors);
+for (i = 0; i < mentors.length; i++) {
```

Comment:
Nice!

You might also like to try `mentors.forEach(printMessageFunction)` where `printMessageFunction` is a function you already defined.

Just remember, only use `forEach` if you want to perform a side effect, like printing something to the console. Where you intend to manipulate an array or produce a new value, use `map`, `filter`, `find` etc. (We'll cover them in more depth next week.